OpenTabletDriver.Configurations/Configurations/RobotPen/T9A.json

```
@@ -0,0 +1,41 @@
+{
+  "Name": "RobotPen T9A",
+  "Specifications": {
+    "Digitizer": {
+      "Width": 282.5,
+      "Height": 208.125,
+      "MaxX": 22600.0,
+      "MaxY": 16650.0
```

```suggestion
      "Width": 295.11,
      "Height": 216.69,
      "MaxX": 22656.0,
      "MaxY": 16636.0
```

The MaxX, and MaxY here are final. I assumed that the device has `1950 LPI` resolution to get the new Width and Height, do they look closer to the "truth"?